config/locales/en.yml

```diff
@@ -1504,12 +1504,18 @@ en:
         body_html: "%{type} '%{name}' access has been <strong>revoked</strong> for User '%{first_name} %{last_name}' (%{email})"
         view_details: "To view all the members of %{type} '%{name}', click the button below:"
     pipeline_mailer:
-      complete_email:
+      complete_user_email:
         subject: Pipeline Completed - %{id}
         body_html: Your pipeline <strong>%{id}</strong> has completed successfully.
-      error_email:
+      complete_manager_email:
+        subject: Pipeline Completed - %{id}
+        body_html: Pipeline <strong>%{id}</strong> has completed successfully.
+      error_user_email:
         subject: Pipeline Errored - %{id}
         body_html: Your pipeline <strong>%{id}</strong> has errors that need to be addressed.
+      error_manager_email:
+        subject: Pipeline Errored - %{id}
+        body_html: Pipeline <strong>%{id}</strong> has errors that need to be addressed.
```

Comment:
Can we update this text to `Pipeline <strong>%{id}</strong> did not complete successfully due to errors.`

Reply:
8380cdb721f7a566b26146e5b7ed12a3895e79f2